src/System.Private.CoreLib/shared/System/Diagnostics/DebugProvider.Unix.cs

```diff
@@ -72,12 +73,15 @@ private static void WriteToDebugger(string message)
             }
         }
 
-        private static void WriteToStderr(string message)
+        private static void WriteToStderr(string? message)
         {
+            if (message == null)
+                return;
```

Comment:
I don't think this is necessary, per my previous comments.  Is there a way a null could get to here?